.github/workflows/build.yml

```diff
@@ -153,31 +150,33 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php: ['7.2', '7.3', '7.4', '8.0', '8.1', '8.2', '8.3', '8.4']
+        php: ['8.0', '8.1', '8.2', '8.3', '8.4']
     steps:
       - uses: actions/checkout@v4
+
       - name: Setup PHP
         uses: shivammathur/setup-php@v2
         with:
           php-version: "${{ matrix.php }}"
           ini-values: "phar.readonly=0,zend.exception_ignore_args=Off"
           coverage: none
 
+      - name: Remove vimeo/psalm until it supports PHP 8.4
+        if: matrix.php == '8.4'
+        run: composer remove vimeo/psalm --no-update --dev
+
       - name: Install dependencies
         run: composer install
 
-
       - uses: actions/download-artifact@v4
         with:
           name: behat.phar
 
       - name: Check content
-        run: |
-          ls -R .
+        run: ls -R .
 
       - name: Test the PHAR
-        run: |
-          php ./behat.phar --version
+        run: php ./behat.phar --version
```

Comment:
Please revert this reformatting to keep this commit diff focused on the actual changes. Likewise the added & removed empty lines elsewhere in the file.

Reply:
I do not see these things as issues as they are improving the overall readability.